src/components/LessonList/components/PaginatedLessonList/components/LessonTeaser/index.js

```diff
@@ -35,14 +34,14 @@ export default ({instructor, lesson}) => {
         }}>
           {lesson.summary
             ? <Markdown>
-                {truncate(lesson.summary, {length: 144})}
+                {lesson.summary}
               </Markdown>
-            : '...'
+            : null
           }
         </div>
         <div className='mt2'>
           <Link to={`/lessons/${lesson.slug}`}>
-            <Button size='small'>
+            <Button size='extra-small'>
```

Comment:
We had some gigantic buttons. They made my balance and composition OCD trigger like crazy. We expect design support soon so this is just temporary.